_data/toc.yaml

```diff
@@ -464,78 +464,10 @@ guides:
   - path: /compliance/cis/
     title: CIS
 
-- sectiontitle: Open source at Docker
+- sectiontitle: Contributors guide
   section:
-  - path: /opensource/code/
```

Comment:
With taking this stuff away, do we need to make some kind of redirect that redirects `/opensource/*` to `/opensource/`? @johndmulhausen